vetiver/server.py

```diff
@@ -100,6 +100,10 @@ def pin_url():
         async def ping():
             return {"ping": "pong"}
 
+        @app.get("/metadata")
```

Comment:
This was, in fact, pretty quick!  But now I wonder if we need to expose prototypes as well?

Reply:
I think that would be a nice addition, while we're working on the API, but let's add it with an argument (for folks with high data privacy needs). I think opt-out (so defaulting to _adding_ the prototype endpoint) is OK for this.